connect-examples/v2/node_bookings/public/stylesheets/datepicker.css

```diff
@@ -0,0 +1,188 @@
+/* Overrides some of the styling for JQuery datepicker */
+.ui-datepicker {
+  width: 672px;
+  padding: .2em .2em 0;
+  display: none;
+  margin-bottom: 40px;
+}
+
+.ui-widget-header {
+  background: white;
+  border: none;
+  border-bottom: 1px solid #EBEBEB;
+}
+
+.ui-datepicker .ui-datepicker-header {
+  position: relative;
+  padding: .2em 0;
+  width: 672px;
+}
+
+.ui-datepicker th {
+  padding: 25px 20px;
+}
+
+.ui-datepicker .ui-datepicker-prev,
+.ui-datepicker .ui-datepicker-next {
+  position: absolute;
+  top: 10px;
+  width: 40px;
+  height: 40px;
+}
+
+.ui-datepicker-prev span,
+.ui-datepicker-next span {
+  background-image: none !important;
+}
+
+.ui-datepicker-prev:before,
```

Comment:
[Psuedo-elements](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements) like `::after` and `::before` should always have double `::` instead of `:`.
[Psuedo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes) like `:hover` only have one `:`.